<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QMainWindow" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="MyLogo">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="LMyName">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTextBrowser" name="MsgArea">
          <property name="acceptRichText">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="DoSend">
            <property name="minimumSize">
             <size>
              <width>101</width>
              <height>31</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>101</width>
              <height>31</height>
             </size>
            </property>
            <property name="text">
             <string>发送消息</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextEdit" name="MsgEdit">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>120</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="UListWidget">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::MultiSelection</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>控制</string>
    </property>
    <addaction name="ActMinSize"/>
    <addaction name="ActTray"/>
    <addaction name="separator"/>
    <addaction name="ActQuit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>好友</string>
    </property>
    <addaction name="ActRefresh"/>
    <addaction name="ActChat"/>
    <addaction name="separator"/>
    <addaction name="ActUserInfo"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>聊天</string>
    </property>
    <addaction name="ActSend"/>
    <addaction name="ActClear"/>
    <addaction name="separator"/>
    <addaction name="ActChatLog"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>开发人员</string>
    </property>
    <addaction name="separator"/>
    <addaction name="ActDebugOut"/>
    <addaction name="ActOptionFile"/>
    <addaction name="separator"/>
    <addaction name="ActSource"/>
    <addaction name="ActProtocol"/>
    <addaction name="ActDevBBS"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>帮助|关于</string>
    </property>
    <addaction name="ActAbout"/>
    <addaction name="ActMember"/>
    <addaction name="ActABoutQt"/>
    <addaction name="ActVer"/>
    <addaction name="separator"/>
    <addaction name="ActJyboxIndex"/>
    <addaction name="ActCheckNew"/>
    <addaction name="ActUserBBS"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="ActMinSize">
   <property name="text">
    <string>最小化</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="ActTray">
   <property name="text">
    <string>隐藏到任务栏</string>
   </property>
   <property name="toolTip">
    <string>隐藏到任务栏</string>
   </property>
  </action>
  <action name="ActQuit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="ActRefresh">
   <property name="text">
    <string>刷新在线列表</string>
   </property>
  </action>
  <action name="ActChat">
   <property name="text">
    <string>打开私聊窗口</string>
   </property>
  </action>
  <action name="ActUserInfo">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Act/images/web.png</normaloff>:/Act/images/web.png</iconset>
   </property>
   <property name="text">
    <string>查看选中好友的资料</string>
   </property>
  </action>
  <action name="ActSend">
   <property name="text">
    <string>发送消息</string>
   </property>
   <property name="shortcut">
    <string>Alt+S</string>
   </property>
  </action>
  <action name="ActClear">
   <property name="text">
    <string>清屏</string>
   </property>
  </action>
  <action name="ActDebugOut">
   <property name="text">
    <string>调试输出</string>
   </property>
   <property name="toolTip">
    <string>调试输出</string>
   </property>
  </action>
  <action name="ActOptionFile">
   <property name="text">
    <string>配置文件</string>
   </property>
  </action>
  <action name="ActSource">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Act/images/web.png</normaloff>:/Act/images/web.png</iconset>
   </property>
   <property name="text">
    <string>项目开发主页(源代码)</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="ActProtocol">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Act/images/web.png</normaloff>:/Act/images/web.png</iconset>
   </property>
   <property name="text">
    <string>0.1版通讯协议</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="ActDevBBS">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Act/images/web.png</normaloff>:/Act/images/web.png</iconset>
   </property>
   <property name="text">
    <string>开发人员论坛</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="ActAbout">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Act/images/web.png</normaloff>:/Act/images/web.png</iconset>
   </property>
   <property name="text">
    <string>零毫秒是什么？</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="ActMember">
   <property name="text">
    <string>合作开发人员名单</string>
   </property>
  </action>
  <action name="ActABoutQt">
   <property name="text">
    <string>感谢Qt</string>
   </property>
  </action>
  <action name="ActVer">
   <property name="text">
    <string>版本信息</string>
   </property>
  </action>
  <action name="ActJyboxIndex">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Act/images/web.png</normaloff>:/Act/images/web.png</iconset>
   </property>
   <property name="text">
    <string>精英盒子主页</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="ActCheckNew">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Act/images/web.png</normaloff>:/Act/images/web.png</iconset>
   </property>
   <property name="text">
    <string>检查新版本</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="ActUserBBS">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Act/images/web.png</normaloff>:/Act/images/web.png</iconset>
   </property>
   <property name="text">
    <string>用户论坛</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="ActChatLog">
   <property name="text">
    <string>聊天记录</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
